problem_builder/tests/unit/test_answer_mixin.py

```diff
@@ -25,7 +25,7 @@ def setUp(self):
 
     def make_answer_mixin(self, name=None, course_id=None, student_id=None):
         if name is None:
-            name = get_random_string()
+            name = get_random_string(12)
```

Comment:
why is this change needed?

Reply:
https://github.com/django/django/blob/f474ba4cb52a41a62da6d6fb3492a95eeb8b3999/django/utils/crypto.py#L51

bit length is required argument for `get_random_string` in 4.2